### ISSUE TITLE
A11y_SQLAzureDataStudio_Notebook_Create new notebook_ScreenReader: There is no announcement of the output cleared.

### ISSUE THREAD

**Issue by v-atkulk** [microsoft/azuredatastudio#9404]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11y_SQLAzureDataStudioBenchmark; #A11yWCAG2.1; #A11ySev2; #A11yTCS; #SQL Azure Data Studio; #Benchmark; #DesktopApp; #MacOs; #MAS4.1.3; #VoiceOver;

Environment Details:
Application Name: Azure Data Studio
Version: 1.15.0-insider
Commit: 3a8b74a
Date: 2020-02-11T05:49:41.181Z
VS Code: 1.42.0
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Darwin x64 19.3.0
Mac OS:10.15.3

Additional Details:
Screen reader: VoiceOver
Mac OS:10.15.3
MAS References:MAS4.1.3

Repro Steps:
1.Launch Azure Data Studio.
2.Click File then New Notebook.
3.Launch screen reader as VoiceOver.
4.Use tab and navigate to clear results button.

**Actual:**
When the code has been executed hit enter on clear results button the data is cleared but no announcement of the same.
Similar for run cell button

**Expected:**
There should be an announcement stating the data has been cleared.

**User Impact:**
Screen reader users will not know if the data has been cleared or the cell is running.

**Attachment link for Reference:**
[9404_A11y_SQLAzureDataStudio_Notebook_Create new notebook_ScreenReader.zip](https://github.com/microsoft/azuredatastudio/files/4297960/9404_A11y_SQLAzureDataStudio_Notebook_Create.new.notebook_ScreenReader.zip)

**Comment by alanrenmsft** [comment]
not sure whether we need to announce the result of all the actions, if so we have a bigger problem here, need to review all the actions in the notebook feature.

**Comment by alanrenmsft** [comment]
I will ask the accessibility team for guideline

**Comment by Charles-Gagnon** [comment]
Discussed this in office hours with v-tayazd - for actions which execute immediately such as these we shouldn't be announcing that an action is occurring as that tends to be too verbose.

**Comment by fsteffi** [comment]
GithubTags:#A11y_SQlADS_WCAG2.1;

**Comment by manaswita-chougule** [comment]
As per the above comments, adding the closing tag to bug.
#Closed;

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-MAC-Feb2020;
